android/src/main/java/io/ably/flutter/plugin/AblyMethodCallHandler.java

```diff
@@ -166,6 +174,88 @@ private void connectRealtime(@NonNull MethodCall call, @NonNull MethodChannel.Re
         });
     }
 
+    private void attachRealtimeChannel(
+            @NonNull MethodCall call, @NonNull MethodChannel.Result result
+    ) {
+        final AblyFlutterMessage message = (AblyFlutterMessage)call.arguments;
+        this.<AblyFlutterMessage<Map<String, Object>>>ablyDo(message, (ablyLibrary, ablyMessage) -> {
+            try {
+                String channelName = (String) ablyMessage.message.get("channel");
+                ChannelOptions channelOptions = (ChannelOptions) ablyMessage.message.get("options");
+                ablyLibrary
+                        .getRealtime(ablyMessage.handle)
```

Comment:
IMO, code-wrapping always must be double indented to differentiate from regular code.


Found [a reference from Oracle docs](https://www.oracle.com/java/technologies/javase/codeconventions-indentation.html) pretty close to this.

Reply:
hmmm... that page lost me at "Avoid lines longer than 80 characters, since they're not handled well by many terminals and tools." 😆  And then I saw that it was last updated in 1999. So, yeah, I would infer standards from that piece with caution!

But, anyway, it's not that important so let's just leave it.

I'll bring checkstyle in soon enough... mwah, hah hah!